README.md

```diff
@@ -1,17 +1,14 @@
 # DataJoint Element - Lab
```

Comment:
Should we provide a link to the `Background.md` file in the `README.md`?

Reply:
Good point.